### ISSUE TITLE
img/users/undefined not found in new Apache-based installation.

### ISSUE THREAD

**Issue by tekhnee** [tableaunoir/tableaunoir#139]
Dear @francoisschwarzentruber, experimenting with a self-hosted instance on Apache (unfortunately ngix is currently not an option).

The frontend appears to work correctly so far, except for the following resource, which fails to load:

`img/users/undefined`

The GET request is initiated by `UserListComponent.ts:52`.

Any thoughts as to why this might be happening?

In my Apache virthalhost, I have mapped my tableaunoir domain to the `./dist/ directory as follows:

```
        DocumentRoot /var/www/html/tableaunoir/dist
        DirectoryIndex index.html
```

I also have reverse proxy and WebSocket settings in place — but suppose they are irrelevant to this request.

Many thanks for any pointers.

**Comment by francoisschwarzentruber** [comment]
Dear @tekhnee,
thank you for your message. Indeed! This was a bug. Normally, you should not have the error anymore.

**Comment by tekhnee** [comment]
Thank you for your speedy response, @francoisschwarzentruber.

I'm afraid that the issue persists on my server — hopefully not an embarrassing mistake on my part.

I did rebuild the project after pull your latest revisions (the new favicon is displayed, for instance). I also cleared the browser cache, restarted `apached` (should be unnecessary, but just to be sure), and even restarted the tableaunoir server (ditto).
